worldedit-core/src/main/java/com/sk89q/worldedit/internal/block/BlockStateIdAccess.java

```diff
@@ -67,7 +69,9 @@ public static int getBlockStateId(BlockState holder) {
         //return blockStateInternalId.getInternalId(holder);
     }
 
-    public static @Nullable BlockState getBlockStateById(int id) {
+    @Nullable
+    public static BlockState getBlockStateById(int id) {
+        int test = id / 10000;
```

Comment:
what is this line for?

Reply:
No idea. It must be remains from an upstream merge.